gh-actions/common/build-debian/action.yml

```diff
@@ -148,6 +158,16 @@ runs:
           if [ -n "${GITHUB_TOKEN}" ]; then
             git config --system url."https://api:${GITHUB_TOKEN}@github.com/".insteadOf "https://github.com/"
           fi
+          # END of FIXME
+
+          if [ -n "${{ inputs.extra-source-build-script != '' && 'true' || '' }}" ]; then
```

Comment:
Huh, I didn't know that GitHub allows this kind of expression evaluation, but it seems like it does: https://docs.github.com/en/actions/writing-workflows/choosing-what-your-workflow-does/evaluate-expressions-in-workflows-and-actions#operators

TIL!

Reply:
Yeah, it's useful here, because otherwise I'd have to care about bash escaping, that wouldn't really work well unless using extra magic... Reason why I preferred this strategy :)